Cargo.toml

```diff
@@ -36,86 +39,128 @@ etherparse = "0.13.0"
 chrono = { version = "0.4.26", default_features = false, features = ["clock"] }
 indexmap = "1.9.3"
 plotters = { version = "0.3.4", default_features = false, features = ["area_series"] }
-iced = { version = "0.9.0", features = ["tokio", "svg"] }
+iced = { version = "0.9.0", features = ["tokio", "svg", "image"] }
 iced_native = "0.10.3"
 iced_lazy = "0.6.1"
 plotters-iced = "0.8.0"
 maxminddb = "0.23.0"
 confy = "0.5.1"
 serde = { version = "1.0.163", default_features = false, features = ["derive"] }
 rodio = { version = "0.17.1", default_features = false, features = ["mp3"] }
-reqwest = { version = "0.11.18", features = ["json", "blocking"] }
+reqwest = { version = "0.11.18", default-features = false, features = ["json", "blocking", "rustls-tls"] }
 dns-lookup = "2.0.2"
 
 #───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
 
 [dev-dependencies]
 rstest = "0.17.0"
 
+#───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+
+[target."cfg(windows)".build-dependencies]
+winres = "0.1.12"
```

Comment:
given that the icon problem seem to not being present on most platforms, I'd like to remove all the modifications related to it (winres, build.rs, the png file, comments on main.rs)

Reply:
Maybe I'm just misunderstanding what you mean by icon of the executable

Reply:
No, the executable and all the shortcuts to it were without icons. I believe the icon we set on the WXS template is only used for the installer itself. You might want to double-check this again...

Reply:
I've removed the code related to the window icon...

Reply:
> Hasn't it to do with what I said to you this morning in [#246 (comment)](https://github.com/GyulyVGC/sniffnet/pull/246#discussion_r1219440271) and [#246 (comment)](https://github.com/GyulyVGC/sniffnet/pull/246#discussion_r1219391525)? As I said, even with the old version of the installer I was able to correctly see everything (on taskbar, desktop shortcut, and windows menu).

I considered them separate issues when I was testing the app on Windows, I actually fixed the executable icon first in order to have Desktop & Programs Menu shortcuts to display the app icon. Later on, I noticed the missing icon in the taskbar which I tried to solve by setting the window icon in Iced settings. However, the latter was resolved on my final check and the former can be double-checked again.

Reply:
It's not a problem for me to keep winres, I was just trying to figure out if it gives benefits from the previous situation